scripts/mutect2_wdl/mutect2.json

```diff
@@ -17,5 +17,6 @@
   "Mutect2.dbsnp_index": "$__dbsnp_index__",
   "Mutect2.cosmic": "$__cosmic__",
```

Comment:
his will work without a PoN -- note that in wdl if `pon` is optional, then ${"--normalPanel " + pon}` yields *nothing* is `pon` is not specified.